scripts/download-env.sh

```diff
@@ -425,6 +455,24 @@ echo "Installing HDMI2USB-mode-switch (flashing and config tool)"
 pip install --upgrade git+https://github.com/timvideos/HDMI2USB-mode-switch.git
 check_import_version hdmi2usb.modeswitch $HDMI2USB_MODESWITCH_VERSION
 
+if [ "$FIRMWARE" = "zephyr" ]; then
```

Comment:
These need to be in `enter-env.sh` script too.

You should be able to load `enter-env.sh` and `download-env.sh` in a side-by-side diff tool (like vimdiff) and see the comparisons.....